.github/CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+# Team
+# @luongvo is the Team Lead
+* @suho @luongvo @phongvhd93 @markgravity @lydiasama
+
+# Engineering Leads
+CODEOWNERS @nimblehq/engineering-leads
```

Comment:
It's gonna be okay, right?

![Screenshot 2566-07-25 at 2 41 16 PM](https://github.com/nimblehq/kmm-templates/assets/12026942/7e2339b2-d078-432c-b36d-b6dd45c34f60)

Reply:
let me remove it, we don't have EL in template projects.

Reply:
Fixed in https://github.com/nimblehq/kmm-templates/pull/11/commits/5ad4a3ba6ceb824980cf27c7e013587df8caf097